### PR TITLE
Schedule merge for Prater

### PR DESCRIPTION
**Pull request by ajsutton**
**What does this do?**

Proposes a schedule for merging of Prater and Görli:

* Bellatrx activation epoch 112260 (2022-08-04 12:24 pm UTC, 8:24am NY, 10:24pm Brisbane/Sydney)
* TTD 10790000 as per https://github.com/ethereum/execution-specs/pull/563

**Cute Animal Picture**

Cutest dog in Ethereum, with pleading look
*Plz Ser, Wen Merge?*

### CODE REVIEW

File: shared/prater/config.yaml [hunk 1]
```diff
@@ -3,6 +3,15 @@
 # Extends the mainnet preset
 PRESET_BASE: 'mainnet'
 
+# Transition
+# ---------------------------------------------------------------
+# TBD, 2**256-2**10 is a placeholder
```
Comment (djrtwo) [inline shared/prater/config.yaml:8]: Fix comment, this is no longer a place holder

**Review by rolfyone — APPROVED** [review]
LGTM

File: shared/prater/config.yaml (hunk 1 above)
Comment (ajsutton) [reply shared/prater/config.yaml]: Good catch, fixed.

**Review by potuz — APPROVED** [review]
Echoing on Paul's words about the time

**Review by dapplion — APPROVED** [review]

**Comment by dapplion** [comment]
:panda_face: :+1:

**Review by tersec — APPROVED** [review]
:panda_face:

**Review by arnetheduck — APPROVED** [review]

**Review by arnetheduck — APPROVED** [review]

**Review by prestonvanloon — APPROVED** [review]

**Review by parithosh — APPROVED** [review]

**Review by dapplion — APPROVED** [review]

**Pull request merged**